guide/popular-topics/intents.md

```diff
@@ -1,97 +1,69 @@
 
-v12 of the library uses Discord API v6, which does not require intents to be specified when connecting to Discord. Specifying intents will be required when the library moves to API v8 in the next major release.
+:::tip
+For up-to-date information about gateway intents (including the full list of available intents and corresponding events) please check the [Discord API Documentation (Gateway Intents)](https://discord.com/developers/docs/topics/gateway#gateway-intents).
 :::
 
-Gateway Intents were introduced to the library in v12 and allow you to pick which events your bot will receive. Intents are groups of pre-defined events that the discord.js client will conditionally subscribe to. For example, omitting the `DIRECT_MESSAGE_TYPING` intent would prevent the discord.js client from receiving any typing events from direct messages. Intents also enable you to remove unwanted data from polluting your bot's cache, however we can not yet explicitly list which unwanted side effects omitting a certain event may have on the internal workings of the library.
+## Privileged Intents
 
-<branch version="11.x">
+Discord defines some intents as "privileged" due to the sensitive nature of the data sent through the affected events. At the time of writing this article privileged intents are `GUILD_PRESENCES` and `GUILD_MEMBERS`. If your bot is not verified and in less than 100 guilds you can enable privileged gateway intents in the [Discord Developer Portal](https://discord.com/developers/applications) under "Privileged Gateway Intents" in the "Bot" section. If your bot is already verified or is about to [require verification](https://support.discord.com/hc/en-us/articles/360040720412) you need to request privileged intents in your verification application or by reaching out to Discord's [support team](https://dis.gd/contact) including the reason why you require access to privileged intents.
 
-Intents are not available in version 11, please update to version 12 of the library if you want to use gateway intents in your bot.
+Before storming off and doing so you should stop and carefully think about if you need these events. Discord made them opt-in so users across the platform can enjoy a higher level of [privacy](https://en.wikipedia.org/wiki/Privacy_by_design). Presences can expose quite a bit of personal information through games and online times, for example. You might find it sufficient for your bot to have a little less information about all guild members at all times, considering you still get the command author as GuildMember from the command execution message and can fetch targets separately.
 
-</branch>
+### Symptoms in version 12
 
-<branch version="12.x">
+- Login times out if you try to fetch all members when starting your bot
```

Comment:
Logins don't timeout on v12 anymore, you should only see a debug log that fetching members timed out (as mentioned a few lines below), and then the client should ready itself as usual:

https://github.com/discordjs/discord.js/blob/8a7abc9f06d44bf693e35a615bb6ba2c3eb1d6e7/src/client/websocket/WebSocketManager.js#L402-L415